azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -39,6 +33,9 @@ CREATE TABLE IF NOT EXISTS image_versions (
   UNIQUE (type_id, version)
 );
 
+CREATE INDEX image_versions_type_id
+  ON image_versions (type_id);
```

Comment:
Do we need index on state column as well for image_versions table?

Reply:
so the type id is a foreign key. Hence it will be useful to have indexed for it. For the state, the index is not required.

Reply:
If there is state column in where clause then it will be useful.

Reply:
the state is not there in the where clause for any query

Reply:
Sounds good. Thank you for clarifying the doubt.